addons/l10n_ar/models/res_currency.py

```diff
@@ -1,9 +1,15 @@
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
-from odoo import fields, models
+from odoo import fields, models, api
 
 
 class ResCurrency(models.Model):
 
     _inherit = "res.currency"
 
     l10n_ar_afip_code = fields.Char('AFIP Code', size=4, help='This code will be used on electronic invoice')
+    l10n_ar_country_code = fields.Char(compute='_compute_l10n_ar_country_code')
+
+    @api.depends_context('company')
+    def _compute_l10n_ar_country_code(self):
+        for record in self:
```

Comment:
@rigr-odoo Mostly I agree, but here it is a common object and as we do not have a good logic for those, better to skip it for the moment.

Reply:
Yep since ResCurrency has no `company_id`, I'm using the one from the environment to hide the AR-fields if we the current company's fiscal country is not Argentina.
Your suggestion would be to let this field visible even if we are in a company whose fiscal country is not Argentina ?

Reply:
yes, we have the same problem for products and partners and this is also not the object where it will be the most disturbing. @rigr-odoo

Reply:
We'll move `country_code` into the `base` module has discussed with @epictete and @william-andre and also do it for the `FiscalPosition` :)